### PR TITLE
VIH-7804 VHO can see staff member in command center

### PR DESCRIPTION
**Pull request by AWildBugAppeared**
https://tools.hmcts.net/jira/browse/VIH-7804

This PR is to allow staff members to be viewed in the command center by the VHO, and to have the same statuses as the judge.

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <p id="tooltip-joined-by-quick-link-{{ participant.id }}" class="govuk-!-margin-bottom-1">{{ joinByQuickLinkText }}</p>
   </div>
   <div class="govuk-grid-column-full">
-    <p *ngIf="participant.showCaseRole" id="tooltip-case-role-{{ participant.id }}" class="govuk-!-margin-bottom-1">
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html:17]: What is the purpose of having data-case-role as an attribute? It appears to just be used in the test as a selector is that correct?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        spyOnProperty(component.participant, 'showCaseRole', 'get').and.returnValue(true);
+        fixture.detectChanges();
+
+        const showCaseRoleElement = fixture.debugElement.query(By.css('[data-case-role]'));
+
+        expect(showCaseRoleElement).toBeTruthy();
+    });
+
+    it('should show case type group', () => {
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.spec.ts:38]: should _not_ show?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.spec.ts [hunk 3]
```diff
@@ -245,4 +249,32 @@ describe('ParticipantStatusComponent', () => {
...
+        hostRoles.forEach(role => {
+            it(`should return get status for ${role} when not in a hearing`, () => {
+                const participant = participants[0];
+                participant.role = role;
+                participant.host_in_another_hearing = false;
+                const participantContactDetails = new ParticipantContactDetails(participant);
+
+                component.setParticipantStatus(ParticipantStatus.InHearing, participantContactDetails);
+
+                expect(participantStatusReaderSpy.getStatusAsTextForHost).toHaveBeenCalled();
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.spec.ts:276]: Perhaps give this an and.returnValue and test the statusText is actually set correctly. See line 160

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html [hunk 4]
```diff
@@ -14,7 +14,7 @@
...
+    <p *ngIf="participant.showCaseRole" id="tooltip-case-role-{{ participant.id }}" class="govuk-!-margin-bottom-1" data-case-role>
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html:17]: What is the purpose of having data-case-role as an attribute? It appears to just be used in the test as a selector is that correct?

Comment (FarronF) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html:17]: I would recommend using the id instead in the tests if this is the case.

**Review by PrasannaDommalapati — DISMISSED** [review]
LGTM

**Review by omio-ma — APPROVED** [review]

**Review by FarronF — APPROVED** [review]

**Pull request merged**